docker/compose.yml

```
@@ -0,0 +1,70 @@
+networks:
+  avalanchelocal:
+    ipam:
+      driver: default
+      config:
+        - subnet: 177.17.0.0/24
+services:
+  avalanche-local-validator01:
+    image: "ash/avalanche-node-local-validator01:${AVALANCHEGO_VERSION:-1.10.9}-${AVALANCHEGO_VM_NAME:-subnet-evm}-${AVALANCHEGO_VM_VERSION:-0.5.6}"
+    container_name: ash-avalanche-local-validator01
+    ports:
+      - "9650:9650"
+      - "9651:9651"
+    networks:
+      avalanchelocal:
+        ipv4_address: 177.17.0.11
+    environment:
+      SUBNET_ID: "${SUBNET_ID:-''}"
```

```suggestion
      SUBNET_ID: "${SUBNET_ID:-}"
```

`''` is not a valid base58 string, with this the containers fail with:

```
couldn't load node config: couldn't parse subnetID "''": base58 decoding error: Invalid base58 digit ('\'')
```